dagger-core/src/test/java/io/odpf/dagger/core/sink/influx/errors/ValidErrorTest.java

```diff
@@ -1,18 +1,16 @@
 package io.odpf.dagger.core.sink.influx.errors;
 
 import org.influxdb.dto.Point;
-import org.junit.Assert;
 import org.junit.Before;
 import org.junit.Test;
-import org.mockito.Mock;
 
+import java.util.Collections;
+
+import static org.junit.Assert.*;
 import static org.mockito.MockitoAnnotations.initMocks;
 
 public class ValidErrorTest {
-    //TODO static import
-    //TODO remove mock
-    @Mock
-    private Iterable<Point> points;
+    private Iterable<Point> points = Collections.emptyList();
```

Comment:
Either initialize inside setup to avoid any repercussions of multiple tests. Or initialize inside the only test which is using it.